test/cmd/builds.sh

```diff
@@ -113,7 +113,7 @@ os::test::junit::declare_suite_end
 
 os::test::junit::declare_suite_start "cmd/builds/config"
 os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook GitHub.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/github"
-os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook Generic.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/generic"
+os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook Generic.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/generic/false"
```

Comment:
yeah so i would not expect "false" and "true" to be part of the webhook url.  there aren't two webhooks here, one at true and one at false.